src/main/java/com/amazon/opendistroforelasticsearch/security/securityconf/impl/v6/ConfigV6.java

```diff
@@ -81,8 +82,8 @@ public String toString() {
         }
     }
 
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
```

Comment:
Please add unit tests for PATCH around this

Reply:
Please add relevant tests

Reply:
They are already added.